Commands/DownloadGoogleBinaries.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace ShyimWebP\Commands;
+
+use Phar;
+use PharData;
+use RecursiveDirectoryIterator;
+use RecursiveIteratorIterator;
+use Shopware\Commands\ShopwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class DownloadGoogleBinaries extends ShopwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('shyim:webp:download-google-binaries')
+            ->setDescription('Downloads google binaries that can convert images to webp')
+        ;
+    }
+
+    /**
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     * @return int
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $style = new SymfonyStyle($input, $output);
+
+        if (!$this->is64bit()) {
+            $style->error('There are no binaries for non-64-bit systems');
+            return 1;
+        }
+
+        $url = '';
+        $packageDirectory = '';
+
+        if ($this->isLinux()) {
+            $url = 'https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.0.1-linux-x86-64.tar.gz';
+            $packageDirectory = 'libwebp-1.0.1-linux-x86-64';
+        } else if ($this->isMac()) {
+            $url = 'https://storage.googleapis.com/downloads.webmproject.org/releases/webp/libwebp-1.0.1-mac-10.13.tar.gz';
+            $packageDirectory = 'libwebp-1.0.1-mac-10.13';
+        } else {
+            $style->error('Downloading binaries is supported for linux and mac only');
+            return 2;
+        }
+
+        $downloadedPackage = tempnam(sys_get_temp_dir(), 'libwebp') . '.tar.gz';
+
+        copy($url, $downloadedPackage);
```

Comment:
You should remove the downloadedPackage if anything is processed

Reply:
True